requirements.txt

```diff
@@ -1,7 +1,7 @@
 dnspython==2.3.0
 fastapi==0.98.0
 httpx==0.24.1
-langchain==0.0.230
+langchain==0.0.312
```

Comment:
After uprating this package version, did you re-generated the container?
```
docker-compose up --build
```
And tried all our db connections to check that they work?